user-interface/src/lib/components/combobox/ComboBox.test.tsx

```diff
@@ -733,19 +733,25 @@ describe('test cams combobox', () => {
     const ref = React.createRef<ComboBoxRef>();
     renderWithProps({ multiSelect: false }, ref);
 
-    const input = document.querySelector('.combo-box-input');
-    expect(input).toBeInTheDocument();
+    const expandButton = document.querySelector('.expand-button');
+    expect(expandButton).toBeEnabled();
+    fireEvent.click(expandButton!);
 
     const listButtons = document.querySelectorAll('li button');
     fireEvent.click(listButtons![0]);
 
     const pill = document.querySelector('.pill');
-    const expandButton = document.querySelector('.expand-button');
-
-    expect(input).not.toBeInTheDocument();
-    expect(expandButton).toBeEnabled();
     expect(pill).toBeEnabled();
 
+    await waitFor(() => {
+      const input = document.querySelector('.combo-box-input');
+      expect(input).toBeInTheDocument();
+      // TODO: It remains a mystery as to why this isn't working...
```

Comment:
I believe these comments have been resolved, but might want to take a look

Reply:
git blame -- who left the comments?  These are good questions I suppose.